swift/swift.go

```diff
@@ -109,6 +110,21 @@ func (c *Client) GetReader(containerName, objectName string) (io.ReadCloser, htt
 	return requestData.RespReader, requestData.RespHeaders, err
 }
 
+// GetReadHandle creates a handler to the specified object's data.
+func (c *Client) GetReadHandle(containerName, objectName string) (io.ReadSeeker, http.Header, error) {
+	requestData := goosehttp.RequestData{}
+	err := c.touchObject(&requestData, client.HEAD, containerName, objectName)
+	if err != nil {
+		return nil, nil, err
+	}
+	l, err := strconv.ParseInt(requestData.RespHeaders.Get("Content-Length"), 10, 64)
```

Comment:
I agree. I could copy ContentLength from net/http.Response into a new field in goose client. 

I do not understand entire existence of goose client hiding the underlying net/http.Response. It seems requestData's Resp* fields should instead be the net/http.Response. Since I cannot understand its design, I don't know the right fix here. Please someone tell me what to do.